client/src/interface/pages/FileManager/constants.js

```diff
@@ -17,12 +17,14 @@ export const S_IROTH = 0o004;
 export const S_IWOTH = 0o002;
 export const S_IXOTH = 0o001;
 
-export const M_IROWR = 0b100000000;
```

Comment:
I believe the M_IXXXX constants are identical to the S_IXXXX ones so I removed them. With those names we can also keep a reference to the Linux source code.